spec/features/items/index_spec.rb

```diff
@@ -17,9 +17,15 @@
 
       it 'links to each individual item show page' do
         visit "/merchants/#{@merchant.id}/items"
-        click_link(@item1.name)
+        first(:link, @item1.name).click
```

Comment:
Just looked this one up because I hadn't seen it previously.  Another tool from the Capybara toolbox!